[docs] Prohibit opening front-end devtools (#5233)
DevExpress/testcafe: docs/articles/documentation/guides/basic-guides/select-page-elements.md

For this purpose, the TestCafe team and community have developed libraries of dedicated selectors for the most popular frameworks.

> > Front-end development tools (such as React DevTools, or Vue DevTools) can interfere with TestCafe and cause errors. Do not open them when you are running or debugging TestCafe tests.

Reviewer: ```suggestion
> Front-end development tools (such as React DevTools or Vue DevTools) can interfere with TestCafe and cause errors. Do not open them while you are run or debug TestCafe tests.
```

### React

> The React selectors module provides the `ReactSelector` class that allows you to select DOM elements by the component name. You can get a root element or search through the nested components or elements. In addition, you can obtain the component props and state.

Reviewer: ```suggestion
The React selectors module provides the `ReactSelector` class, which allows you to select DOM elements by the component name. You can get a root element or search through the nested components or elements. In addition, you can obtain the component props and state.
```
